examples/gradle-and-jvm/src/main/proto/addressbook.proto

```diff
@@ -25,6 +25,11 @@ message Person {
     repeated PhoneNumber phones = 4;
 
     google.protobuf.Timestamp last_updated = 5;
+
+    oneof contactType {
+        string primary = 6;
+        string secondary = 7;
```

Comment:
This `addressbook.proto` file actually comes from the protobuf library's `examples/addressbook.proto` file. I'd prefer to keep the two in sync. We can always add more examples if we need to demonstrate any of the more advanced features.

Reply:
agreed. we can definitely use gitquery to sync that and the conformance files now that i look at them closer. And we dont need the oneof here as its already being generated as a part of the conformance code.